boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpunet.dts

```diff
@@ -24,6 +24,46 @@
 		zephyr,code-partition = &slot0_partition;
 		watchdog0 = &wdt0;
 	};
+
+	arduino_header: connector {
```

Comment:
This connector is defined in `nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi`. Why not move that definition to `nrf5340_audio_dk_nrf5340_shared.dtsi` instead of duplicating it here?

Reply:
Hello @anangl,
I missed this, thanks for the comment! I will update the PR.